middle_end/flambda2/simplify/unboxing/unboxing_epa.ml

```diff
@@ -479,3 +479,81 @@ let add_extra_params_and_args extra_params_and_args decision =
       EPA.add extra_params_and_args ~extra_param ~extra_args:epa.args
   in
   aux extra_params_and_args decision
+
+(* Extracting just the extra params *)
+(* ******************************** *)
+
+let compute_extra_params_in_unspecified_order extra_params decision =
```

Comment:
`extra_params` here don't have to be that. I think that calling that variable `acc` would make it more obvious that this is more 'neutral' to the function.